modules/ducktests/tests/ignitetest/services/utils/ignite_aware_app.py

```diff
@@ -106,7 +108,23 @@ def env(self):
         return "export MAIN_CLASS={main_class}; ".format(main_class=self.servicejava_class_name) + \
                "export EXCLUDE_TEST_CLASSES=true; " + \
                "export IGNITE_LOG_DIR={log_dir}; ".format(log_dir=self.PERSISTENT_ROOT) + \
-               "export USER_LIBS=%s:/opt/ignite-dev/modules/ducktests/target/*; " % self.user_libs
+               "export USER_LIBS=%s:/opt/ignite-dev/modules/ducktests/target/*%s; " % \
```

Comment:
Looks like user_libs and service_libs should be lists and shouldn't know about colons delimeter. As colon is a part of USER_LIBS, now we have usage of the delimiter in 3 places (user_libs, service_libs, USER_LIBS)

Reply:
Done